### PR TITLE
Make new library mobile friendly

### PR DESCRIPTION
**Pull request by pzalep1**
This PR makes the new library mobile friendly and it also inspects the new design for accessibility. 

GO TO THE DESIGN CHANNEL TO SEE ALL THE ANIMATIONS!!!

Screen Shot 2020-03-04 at 11 57 50 AM
Screen Shot 2020-03-04 at 11 58 20 AM
Screen Shot 2020-03-04 at 11 58 34 AM
Screen Shot 2020-03-04 at 11 58 43 AM
Screen Shot 2020-03-04 at 10 49 40 AM

### CODE REVIEW

**Comment by Olaniyan99** [comment]
Hey Paige, are we still adding a notification badge count on the mobile view to correspond to the stacked cards? Because i remember us talking about that.

**Comment by pzalep1** [comment]
> Hey Paige, are we still adding a notification badge count on the mobile view to correspond to the stacked cards? Because i remember us talking about that.

We need the changes made in the other branch with Emily's to get the count. We will add that on her branch once we get everything merged.

File: src/app/cube/library/components/notification-card/notification-card.component.html [hunk 1]
```diff
@@ -1,9 +1,10 @@
 <div class='notification-card'>
-  <button aria-label='delete notification' class='notification-card__exit-button' (activate)='onDeleteButtonClick($event)'>
+  <button aria-label='Dismiss notification' class='notification-card__exit-button' (activate)='onDeleteButtonClick($event)' tip="Dismiss notification"
+  tipPosition="top">
     <i class="fal fa-times"></i>
   </button>
   <h4 class='notification-card__title'>
-    {{ (notification.attributes?.name.length>30) ? (notification.attributes?.name | slice:0:30)+'...' : notification.attributes?.name }}
+    {{ (notification.attributes?.name.length>25) ? (notification.attributes?.name | slice:0:22)+'...' : notification.attributes?.name }}
```
Comment (EVogel1999) [inline src/app/cube/library/components/notification-card/notification-card.component.html]: Why are these different values?

File: src/app/cube/library/library.component.html [hunk 2]
```diff
@@ -60,10 +60,16 @@ <h2 class='library-items__title'>
                         (titleClicked)='goToItem(item)'
                     ></clark-library-item>
                 </div>
+                <div *ngIf='!libraryItems?.length' class='empty-notification-container'>
```
Comment (EVogel1999) [inline src/app/cube/library/library.component.html]: This if statement is nested in the if that states: libraryItems?.length.  Doesn't that mean this is unreachable?

Comment (EVogel1999) [reply src/app/cube/library/library.component.html]: We should maybe use more templates too if we can, to break things up easier.

File: src/app/cube/library/library.component.ts [hunk 3]
```diff
@@ -70,17 +143,79 @@ export class LibraryComponent implements OnInit, OnDestroy{
       });
       this.loading = false;
     } catch (e) {
+      console.log(e);
```
Comment (EVogel1999) [inline src/app/cube/library/library.component.ts]: console.error

File: src/app/cube/library/library.component.ts [hunk 4]
```diff
@@ -70,17 +143,79 @@ export class LibraryComponent implements OnInit, OnDestroy{
...
+  async setNotifications(index: number) {
+    if (index === this.firstIndex) {
+      this.lastIndex = this.notificationCardCount;
+    } else if (index < this.firstIndex && index >= 0) {
+      this.goBackNotifications();
+    } else if (index > this.firstIndex) {
+      await this.goForwardNotifications();
+    }
+    this.notifications = this.localNotifications.slice(this.firstIndex, this.lastIndex);
+    console.log(this.firstIndex);
```
Comment (EVogel1999) [inline src/app/cube/library/library.component.ts]: Console log

File: src/app/cube/library/library.component.ts [hunk 5]
```diff
@@ -92,6 +227,7 @@ export class LibraryComponent implements OnInit, OnDestroy{
     try {
       await this.libraryService.removeFromLibrary(this.libraryItemToDelete.cuid);
       this.libraryItems = (await this.libraryService.getLibrary(1, 10)).cartItems;
+      this.changeLibraryItemPage(this.currentPageNumber);
       this.showDeleteLibraryItemModal = false;
     } catch (e) {
       console.log(e);
```
Comment (EVogel1999) [inline src/app/cube/library/library.component.ts]: Know you didn't write this but, console.error

File: src/app/cube/library/components/notification-card/notification-card.component.html (hunk 1 above)
Comment (pzalep1) [reply src/app/cube/library/components/notification-card/notification-card.component.html]: I changed something in the css to better fit the cards in the container when it resizes and needed five less characters to make it work.

File: src/app/cube/library/library.component.html (hunk 2 above)
Comment (pzalep1) [reply src/app/cube/library/library.component.html]: I will refactor this today

**Pull request merged**